emaki.rb

```diff
@@ -57,17 +57,23 @@
 #
 
 # おまじない
-adapter = DataMapper.setup(:default, adapter: 'redis')
-adapter.resource_naming_convention = lambda do |value|
-  [
-    'emaki',
-    EMAKI_ENV,
-    DataMapper::Inflector.pluralize(
-      DataMapper::Inflector.underscore(value)).gsub('/', '_')
-  ].join(':')
+db_name = 'emaki'
+if EMAKI_ENV == 'test'
```

Comment:
__[rubocop]__
* Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.